src/controllers/accomodation.controller.js

```diff
@@ -105,6 +105,89 @@ class Accommodation {
     const query = await accomodationServices.bookAccommodation(req.user.id, req.body.accommodationId, req.body.roomId, req.body.departureDate, req.body.checkoutDate);
     return response.successMessage(res, 'Booking was successfully processed', 201, query);
   }
+
+  /**
+   * This function helps user to rate accommodation which he booked.
+   * @param {Object} req request
+   * @param {*} res response
+   * @returns {Object} return accommodation response
+   */
+  static async rateAccomodation(req, res) {
+    try {
+      const { rate } = req.body;
+      const userId = req.user.id;
+      const { accomodationId } = req.params;
+
+      const rateData = { rate, userId, accomodationId };
+
+      const rateAccomodation = await accomodationServices.CreateAccomodationRate(rateData);
```

Comment:
This variable name is similar to the function name. The variable name should convey the meaning of what it is holding whereas the function name conveys the meaning of what it is doing.

Reply:
I fixed it I changed the naming of that variables.